format/formatter.go

```diff
@@ -80,9 +80,18 @@ func (f *Formatter) Apply(ctx context.Context, files []*walk.File, filter bool)
 	}
 
 	// execute the command
-	cmd := exec.CommandContext(ctx, f.config.Command, args...)
+	cmd := exec.CommandContext(ctx, f.executable, args...)
 	cmd.Dir = f.workingDir
 
+	// log out the command being executed, truncating it as required
+	if f.log.GetLevel() == log.DebugLevel {
+		cmdStr := cmd.String()
+		if len(cmdStr) >= 1024 {
+			cmdStr = cmdStr[:1024] + "..."
+		}
+		f.log.Debugf("executing: %s", cmd.String())
```

Comment:
```suggestion
		f.log.Debugf("executing: %s", cmdStr)
```
So it gets used :)

Reply:
Is that the command-line arguments, or the output of the execution? If it's the former, I would argue to keep it whole instead of truncating.